pyuvsim/tests/test_simsetup.py

```diff
@@ -375,13 +394,22 @@ def test_healpix_hdf5_catalog():
     """
     pytest.importorskip('astropy_healpix')
     path = os.path.join(SKY_DATA_PATH, 'healpix_disk.hdf5')
-    sky = pyradiosky.SkyModel()
+    sky = SkyModel()
```

Comment:
is there a reason we're not using `.from_file()` here?

Reply:
Yes, there is. This is using an old, non-skyh5 file. We currently still have a reader for that in SkyModel, but it is deprecated and therefore it's not supported via the generic read or the `from_file` method. I considered removing this test, but I left it in for now since the support for these files does still exist in pyradiosky. But I could be convinced to remove it.

Reply:
Ah ok cool.